cfavml/src/danger/impl_avx2fma.rs

```diff
@@ -119,6 +119,62 @@ impl SimdRegister<f32> for Avx2Fma {
     }
 }
 
+impl Hypot<f32> for Avx2Fma {
+    //b' * sqrt((a*b_hat)^2 + (b*b_hat)^2), where |b| > |a|, b'=(2^n<b), b_hat=1/b'
+    #[inline(always)]
+    unsafe fn hypot(x: Self::Register, y: Self::Register) -> Self::Register {
+        // convert the inputs to absolute values
+        let (x, y) = (
+            _mm256_andnot_ps(_mm256_set1_ps(-0.0), x),
+            _mm256_andnot_ps(_mm256_set1_ps(-0.0), y),
+        );
+        // find the max and min of the two inputs
+        let (hi, lo) = (
+            <Avx2 as SimdRegister<f32>>::max(x, y),
+            <Avx2 as SimdRegister<f32>>::min(x, y),
+        );
```

Comment:
Same vibe as mentioned earlier, best to have these just as two separate lines rather than a tuple.